scripts/abi_check.py

```diff
@@ -31,8 +31,8 @@
 class AbiChecker(object):
 
     def __init__(self, report_dir, old_repo, old_rev, old_crypto_rev,
-                 new_repo, new_rev, new_crypto_rev, keep_all_reports, brief,
-                 skip_file=None):
+                 old_crypto_repo, new_repo, new_rev, new_crypto_rev,
```

Comment:
Commit "Extend crypto submodule selection further to choose repo" could use a better explanation of what's going on.

Reply:
OK